fillers/example/acl_example.py

```diff
@@ -0,0 +1,75 @@
+"""
+Test ACL Transaction Source Code Examples
+"""
+
+from ethereum_test_tools import (
+    AccessList,
+    Account,
+    Environment,
+    StateTest,
+    Transaction,
+    test_from_until,
+)
+
+
+@test_from_until("berlin", "london")
+def test_access_list(fork):
+    """
+    Test type 1 transaction.
+    """
+    env = Environment(
+        coinbase="0x2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+        difficulty=0x20000,
+        gas_limit=68719476736,
+        number=16777216,
+        timestamp=4,
+    )
```

Comment:
It seems like the values `difficulty=0x20000` and `timestamp=4` are impossible for block 1 (`env` describes Block 1 in state tests), since the genesis block already has the minimum values of `difficulty=0x20000` and `timestamp=0`, the minimum expected difficulty for block 1 (assuming a timestamp increase of only 4) should be `0x20040`.

Also geth complains about the block number not being 0 for genesis and 1 for the first block (in the case of the resulting block test for this file, the numbers are 16777215 for genesis and 16777216 for the first block).

I filled an issue for this here: https://github.com/ethereum/execution-spec-tests/issues/87

We could discuss the best course of action in the issue, but for the time being I think we should use the default values for `env` in this test.

Reply:
Sure. I just picked these values from another test without much thought to that. I'll make the updates

Reply:
done